Cargo.toml

```diff
@@ -174,3 +174,6 @@ rust_2018_idioms = { level = "warn", priority = -1 }
 # TODO change to deny
 missing_docs = "warn"
 warnings = "warn"
+unexpected_cfgs = { level = "warn", check-cfg = [
```

Comment:
Doesn't this level need to be set to "ignore" or something like that?

Reply:
It's "allow"

Reply:
we also need to change this in sequencer

Reply:
Doesn't seem to help

Reply:
Actually "warn" is fine we were just not adding it in all the Cargo.toml files that needed it.